src/index.html

```diff
@@ -16,17 +17,36 @@
       href="style.css"
     />
   </head>
+  <!-- #endregion -->
+
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      method="post"
+      data-qa="big"
+    >
+      <input
+        type="text"
+        name="big-search"
+        placeholder="Try “Los Angeles”"
+        required=""
+        class="search-bar search-bar__big"
+        data-qa="keypress"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      method="post"
+      data-qa="small"
+    >
+      <input
+        type="text"
+        name="big-search"
```

Comment:
The `name` attribute for this input is the same as the one in the other form. If these inputs are meant to be distinct, consider using unique names to avoid potential conflicts.